includes/admin/plugins.php

```diff
@@ -279,7 +297,7 @@ function perflab_render_plugin_card( array $plugin_data ) {
 		<div class="plugin-card-top">
 			<div class="name column-name">
 				<h3>
-					<a href="<?php echo esc_url( $details_link ); ?>" class="thickbox open-plugin-details-modal">
+					<a href="<?php echo esc_url( $details_link ); ?>"<?php echo wp_kses_post( $title_attr ); ?>>
```

Comment:
Since Kses isn't designed for this context, probably better to just add a `phpcs` comment to disable the rule here, since the variable is essentially a constant.